Aquality.WinAppDriver/tests/Aquality.WinAppDriver.Tests/Applications/Locators/CalculatorWindow.cs

```diff
@@ -17,6 +17,8 @@ public static class CalculatorWindow
 
```

Comment:
Please name branches in accordance to what changes they contain for the next PRs

Reply:
will do